components/event-chart/src/event-chart.spec.ts

```diff
@@ -100,6 +104,12 @@ function getLaneLabels(fixture: ComponentFixture<any>): string[] {
 describe('DtEventChart', () => {
   let overlayContainer: OverlayContainer;
   let overlayContainerElement: HTMLElement;
+  const overlay_config: DtUiTestConfiguration = {
```

Comment:
NIT: please rename to `overlayConfig` according to our coding standards and for sake of consistency.